jsapp/js/components/common/koboSelect.tsx

```diff
@@ -70,6 +71,7 @@ interface KoboSelectProps {
   onChange: (newSelectedOption: string | null) => void;
   'data-cy'?: string;
   placeholder?: string;
+  error?: string;
```

Comment:
This should be `error?: string | undefined;` - a bit counter-intuitive, but you explicitly pass it values that are possibly `undefined` (`props.errors?.country`, `props.errors?.gender`, etc.) instead of just omitting the prop. Should fix some TS errors.

Reply:
Do you happen to have actual TS errors with this? I'm asking because I don't have errors, and I wonder if my setup is not catching the errors :)